src/arc-validate/APIs/ARCSpecification.fs

```diff
@@ -0,0 +1,529 @@
+﻿namespace ARCValidate.ARCSpecification
+
+open ControlledVocabulary
+open ARCTokenization
+open ARCExpect
+open Expecto
+
+module V2_Draft =
```

Comment:
Let's move this into ARCExpect, since that is the place for library stuff, see my other comments